src/main/java/uk/gov/hmcts/reform/sandl.snlevents/controllers/HearingPartController.java

```diff
@@ -71,8 +72,8 @@ public ResponseEntity assignHearingPartToSession(
         @PathVariable UUID hearingPartId,
         @RequestBody HearingPartSessionRelationship assignment) throws IOException {
 
-        HearingPart hearingPart = hearingPartService.assignHearingPartToSession(hearingPartId, assignment);
+        UserTransaction ut = hearingPartService.assignHearingPartToSession(hearingPartId, assignment);
```

Comment:
(minor) I would consider change of the method name so it;s more obvious is going with user transaction....